care/facility/summarisation/patient_summary.py

```diff
@@ -0,0 +1,104 @@
+from django.db.models import Q, Subquery
+from django.utils.timezone import now
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
+
+from rest_framework import serializers
+from rest_framework.generics import get_object_or_404
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.viewsets import GenericViewSet
+
+from care.facility.models import FacilityRelatedSummary, Facility, PatientConsultation
```

Comment:
Codacy found an issue: [Unable to import 'care.facility.models'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5789522)